desktop/components/commercial_filter/filters/keyword/index.jade

```diff
@@ -0,0 +1,13 @@
+#keyword-search-bar-container.typeahead
+  input#keyword-search-bar-input.bordered-input(
+    placeholder='Search…'
+    tabindex='1'
+    itemprop='query-input'
+    type='text'
+    name='keyword'
+    value= term
```

Comment:
In the jade template, I think it might be expecting `term` to be some local accessible to it.

Rather than explicitly putting the value in https://github.com/artsy/force/pull/1037/files#diff-c2205034f41916f2c4a5acfa71190f9dR25, you can try something like:

```
@$el.html template
  term: @params.get('keyword')